app/src/main/java/com/example/triptracker/view/map/MapPopups.kt

```diff
@@ -136,3 +156,177 @@ fun AddressText(mpv: MapPopupViewModel, latitude: Float, longitude: Float) {
 
   Text(text = address, color = Color.White, modifier = Modifier.testTag("AddressText"))
 }
+
+@Preview
+@Composable
+fun DisplayStartScreen() {
+  // test itin
+  val pin1 =
+      Pin(
+          50.05186463055543,
+          14.43129605385369,
+          "HQ",
+          "Jetbrains HQ",
+          listOf(
+              "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/Prague_%286365119737%29.jpg/800px-Prague_%286365119737%29.jpg",
+              "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/Prague_%286365119737%29.jpg/800px-Prague_%286365119737%29.jpg",
+              "https://cdn-vsh.prague.eu/object/254/u-fleku-01.jpg"))
+  val pin2 =
+      Pin(
+          50.0792573623994,
+          14.418225529534855,
+          "U Fleku",
+          "Oldest restaurant",
+          listOf(
+              "https://d3dqioy2sca31t.cloudfront.net/Projects/cms/production/000/033/186/slideshow/aa3cc8bc14fc3f93c823f6aa5743874e/slide-czech-republic-prague-old-town-hall.jpg"))
+  val pin3 = Pin(50.08731011666294, 14.420438033846013, "Clock", "Astronomical Clock", emptyList())
+
+  val loc = Location(50.05186463055543, 14.43129605385369, "HQ")
+  val itin =
+      Itinerary(
+          "1",
+          "Jetbrains Island",
+          "polfuentescam@gmail.com",
+          loc,
+          500,
+          "4",
+          "5",
+          listOf(pin1, pin2, pin3),
+          "test",
+          emptyList())
+  val profile =
+      UserProfile(
+          "pol@gmail.com",
+          "Foo",
+          "Fighter",
+          "29/12/1999",
+          "",
+          "",
+          emptyList(),
+          emptyList(),
+      )
+  StartScreen(itin, profile, UserProfileViewModel(), onClick = {})
+}
+
+@Composable
+fun StartScreen(
+    itinerary: Itinerary,
+    profile: UserProfile,
+    uservm: UserProfileViewModel,
+    onClick: (Pin) -> Unit
+) {
+  // The size of the user's avatar/profile picture
+  val avatarSize = 35.dp
+  var userofpost by remember { mutableStateOf(UserProfile("")) }
+
+  uservm.getUserProfile(itinerary.userMail) { itin ->
+    if (itin != null) {
+      userofpost = itin
+    }
```

Comment:
here you can use the ambient profile

Reply:
i don't think so because it's not the profile of the user but the profile of the person who posted the itinerary